pages/data-fetcher.module.scss

```diff
@@ -0,0 +1,3 @@
+.dataKeyInput {
```

Comment:
let's not put module / css files right in the `pages` folder - consider using the styles folder.

(These are pages, not components)

It will keep the pages folder clear so we can easily see all the routes.